src/index.ts

```diff
@@ -84,7 +84,7 @@ const RESERVED_WORDS = [
 	'else', 'enum', 'extension', 'fallthrough', 'false', 'fileprivate', 'final', 'for', 'func', 'get', 'guard', 'if', 'import', 'in', 'indirect', 'infix', 'init', 'inout',
 	'internal', 'is', 'lazy', 'left', 'let', 'mutating', 'nil', 'none', 'nonmutating', 'open', 'operator', 'optional', 'override', 'postfix', 'precedence', 'prefix', 'private',
 	'protocol', 'public', 'repeat', 'required', 'rethrows', 'return', 'right', 'self', 'set', 'static', 'struct', 'subscript', 'super', 'switch', 'throw', 'throws', 'true',
-	'try', 'typealias', 'unowned', 'var', 'weak', 'where', 'while', 'willSet',
+	'try', 'typealias', 'unowned', 'var', 'weak', 'where', 'while', 'willSet', 'Sendable', 'Hashable',
```

Comment:
This is maybe a slightly odd place to add these... how about we add these in a new line underneath the `LocalDate` family? The line you've added it to finishes the language keywords chunk of reserved words I reckon.

BUT... I reckon we don't add these as reserved words if we don't need to. If you reckon we don't need to add others in here then how about a separate PR to remove those?